include/global/state_space_data.hpp

```diff
@@ -176,9 +176,19 @@ namespace NP {
 				for (const auto& pred : predecessors_suspensions[j.get_job_index()])
 				{
 					auto pred_idx = pred.first->get_job_index();
-					// skip if part of disregard
-					if (contains(disregard, pred_idx))
-						continue;
+
+					Time min_low_suspension = Time_model::constants<Time>::infinity();
+					for (const auto &low_suspension : predecessors_suspensions[j_low]) {
+						if (low_suspension.first->get_job_index() == pred_idx) {
+							if (min_low_suspension == Time_model::constants<Time>::infinity() || low_suspension.second.min() > min_low_suspension) {
+								min_low_suspension = low_suspension.second.min();
+							}
+						}
+					}
+					if (min_low_suspension != Time_model::constants<Time>::infinity()) {
```

Comment:
the `if` can be removed if `min_low_suspension` is initialised to `0` and the loop above takes the max instead of the min on suspension times.